redis/src/connection.rs

```diff
@@ -211,6 +211,9 @@ pub struct ConnectionInfo {
 
     /// A boxed connection address for where to connect to.
     pub redis: RedisConnectionInfo,
+
+    /// In case of reconnection issues, the manager will retry reconnection
+    pub retry_strategy: Option<RetryStrategyInfo>,
```

Comment:
this isn't the place for this - if only one connection type uses this, it should be in that type's config, not here.